src/routes/router.js

```diff
@@ -0,0 +1,11 @@
+const mainRoute = require("./main/main");
```

Comment:
emm, it's basically copy of what Ivan Lapa wrote ) I can't see solution for the hw1 & hw2 here